msteams-platform/resources/schema/localization-schema.md

```diff
@@ -11,8 +11,89 @@ ms.date: 04/14/2025
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.description`|String|128|Specifies a localized value for the [composeExtensions.commands.parameters.description property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].description`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.value`|String|512|Specifies a localized value for the [composeExtensions.commands.parameters.value property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].value`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.parameters\\[[0-4]\\]\\.choices\\[[0-9]\\]\\.title`|String|128|Specifies a localized value for the [composeExtensions.commands.parameters.choices.title property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].parameters[0-4].choices[0-9].title`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.samplePrompts\\[[0-4]\\]\\.text`|String|128|Specifies a localized value for the [composeExtensions.commands.samplePrompts.text property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].samplePrompts[0-4].text`.||
+|`composeExtensions\\[0\\]\\.commands\\[[0-9]\\]\\.taskInfo\\.title`|String|64|Specifies a localized value for the [composeExtensions.commands.taskInfo.title property](manifest-schema.md#composeextensionscommands). The property name should be a JSON path expression in the following form: `composeExtensions[0].commands[0-9].taskInfo.title`.||
+|`activities.activityTypes.description`|String|128|Specifies a localized value for the [activities.activityTypes.description property](manifest-schema.md#activitiesactivitytypes). The property name should be a JSON path expression in the following form: activities.`activityTypes[0-127].description`.||
+|`activities.activityTypes.templateText`|String|128|Specifies a localized value for the [activities.activityTypes.templateText property](manifest-schema.md#activitiesactivitytypes). The property name should be a JSON path expression in the following form: `activities.activityTypes[0-127].templateText`.||
+|`meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name`|String|128|Specifies a localized value for the [meetingExtensionDefinition.scenes.name property](manifest-schema.md#meetingextensiondefinition). The property name should be a JSON path expression in the following form: `meetingExtensionDefinition.scenes[0-4].name`.||
```

Comment:
Is this correct `meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name`?

Because in description we have added like: `meetingExtensionDefinition.scenes[0-4].name`

Reply:
`meetingExtensionDefinition.scenes\\[[0-9]\\]\\.name` is the correct name as that's how it appears in the localizationSchema. `meetingExtensionDefinition.scenes[0-4].name` is just an easier and more legible way of reading it. One thing I did notice is it should be [0-9] instead of [0-4].